src/components/sbb-expansion-panel-content/sbb-expansion-panel-content.tsx

```diff
@@ -0,0 +1,27 @@
+import { Component, ComponentInterface, h, Host, JSX, Prop } from '@stencil/core';
+
+/**
+ * @slot unnamed - Slot to render the content in the sbb-expansion-panel.
+ */
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-expansion-panel-content.scss',
+  tag: 'sbb-expansion-panel-content',
+})
+export class SbbExpansionPanelContent implements ComponentInterface {
+  /**
+   * @internal
+   * Sets the `aria-hidden` attribute on panel when it's collapsed.
+   */
+  @Prop() public expanded: boolean;
```

Comment:
should we maybe solve this with a data-attribute? @kyubisation

Reply:
Indirectly answered in https://github.com/lyne-design-system/lyne-components/pull/1860/files/7455688578e0fdb55480f3bca3b1787497278f18#r1273181935

Reply:
Removed in commit faa41ab